udata/api/__init__.py

```diff
@@ -120,13 +120,19 @@ def search_parser(self, adapter, paginate=True):
         # q parameter
         parser.add_argument('q', type=str, location='args',
                             help='The search query')
+        facets = adapter.facets.keys()
+        if facets:
+            parser.add_argument('facets', type=str, location='args',
+                                choices=['all'] + facets, action='append',
+                                help='Selected facets to fetch')
         # Add facets filters arguments
```

Comment:
I don't know why this comment is not disappearing, I Added some comment in the code.